blockchain/tx_pool.go

```diff
@@ -1029,49 +1043,66 @@ func (pool *TxPool) add(tx *types.Transaction, local bool) (bool, error) {
 		return false, err
 	}
 
-	// If the transaction pool is full and new Tx is valid,
-	// (1) discard a new Tx if there is no room for the account of the Tx
-	// (2) remove an old Tx with the largest nonce from queue to make a room for a new Tx with missing nonce
-	// (3) discard a new Tx if the new Tx does not have a missing nonce
-	// (4) discard underpriced transactions
-	if uint64(pool.all.Slots()+numSlots(tx)) > pool.config.ExecSlotsAll+pool.config.NonExecSlotsAll {
-		// (1) discard a new Tx if there is no room for the account of the Tx
-		from, _ := types.Sender(pool.signer, tx)
-		if pool.queue[from] == nil {
-			logger.Trace("Rejecting a new Tx, because TxPool is full and there is no room for the account", "hash", tx.Hash(), "account", from)
-			refusedTxCounter.Inc(1)
-			return false, fmt.Errorf("txpool is full: %d", uint64(pool.all.Count()))
-		}
```

Comment:
I have question. So before this PR, it sometimes couldn't accept the better priced tx. After PR, it "always" accept the better priced tx. Is my understanding correct?